nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -590,11 +591,15 @@ private WebAppContext loadWar(final File warFile, final String contextPath, fina
 
         // add HTTP security headers to all responses
         final String ALL_PATHS = "/*";
-        ArrayList<Class<? extends Filter>> filters = new ArrayList<>(Arrays.asList(XFrameOptionsFilter.class, ContentSecurityPolicyFilter.class, XSSProtectionFilter.class));
+        ArrayList<Class<? extends Filter>> filters = new ArrayList<>(Arrays.asList(
```

Comment:
I notice the max form content size is set to 600KB on line 590 above. If this is set, doesn't the `10 MB` / `100 MB` limit not apply because Jetty overrides it anyway? 

https://www.eclipse.org/jetty/documentation/current/setting-form-size.html

Reply:
Not a complete solution; see comments on unit test below.